flytekit/configuration/__init__.py

```diff
@@ -344,6 +349,11 @@ def auto(cls, config_file: typing.Optional[typing.Union[str, ConfigFile]] = None
         kwargs = set_if_exists(
             kwargs, "client_credentials_secret", _internal.Credentials.CLIENT_CREDENTIALS_SECRET.read(config_file)
         )
+        kwargs = set_if_exists(
+            kwargs,
+            "client_credentials_secret",
+            read_file_if_exists(_internal.Credentials.CLIENT_CREDENTIALS_SECRET_LOCATION.read(config_file)),
```

Comment:
@wild-endeavor,
Also not sure if this should support relative paths?
And if not, is there a way I can write a test that passes?